### ISSUE TITLE
Specified qdisc not found

### ISSUE THREAD

**Issue by jujoramos** [chaos-mesh/chaos-mesh#1224]
## Bug Report

**What version of Kubernetes are you using?**
```
Client Version: version.Info{Major:"1", Minor:"19", BuildDate:"2020-10-14T12:50:19Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", BuildDate:"2020-08-19T23:48:38Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

**What did you do?**
- Deployed `chaos-mesh` through `helm install chaos-mesh chaos-mesh/chaos-mesh --namespace=chaos-testing --set dashboard.create=false --set prometheus.create=false`.
- Created a simple deployment to test the `network-partition` scenario and deployed the scenario.

**What did you expect to see?**
Network interrupted between `sample-app-1` and the other pods (`sample-app-2` and `sample-app-3`).

**What did you see instead?**
The scenario fails with the `Error: Specified qdisc not found.`.

---

_**ControllerManager Logs**_
```
Controller manager Version: version.Info{GitVersion:"v1.0.2", GitCommit:"1eede0221a0c3ac2add73dec710fecacaad0958c", BuildDate:"2020-11-09T05:56:41Z", GoVersion:"go1.14.6", Compiler:"gc", Platform:"linux/amd64"}
2020-11-26T12:13:51.817Z    INFO    setup    Chaos controller manager is running in cluster scoped mode.
2020-11-26T12:13:52.577Z    INFO    controller-runtime.metrics    metrics server is starting to listen    {"addr": ":10080"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=PodChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-podchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-podchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=PodChaos", "path": "/validate-chaos-mesh-org-v1alpha1-podchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-podchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=NetworkChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-networkchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-networkchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=NetworkChaos", "path": "/validate-chaos-mesh-org-v1alpha1-networkchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-networkchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=IoChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-iochaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-iochaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=IoChaos", "path": "/validate-chaos-mesh-org-v1alpha1-iochaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-iochaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=TimeChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-timechaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-timechaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=TimeChaos", "path": "/validate-chaos-mesh-org-v1alpha1-timechaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-timechaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=KernelChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-kernelchaos"}
2020-11-26T12:13:52.578Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-kernelchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=KernelChaos", "path": "/validate-chaos-mesh-org-v1alpha1-kernelchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-kernelchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=StressChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-stresschaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-stresschaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=StressChaos", "path": "/validate-chaos-mesh-org-v1alpha1-stresschaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-stresschaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-podiochaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.builder    Registering a mutating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=PodNetworkChaos", "path": "/mutate-chaos-mesh-org-v1alpha1-podnetworkchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/mutate-chaos-mesh-org-v1alpha1-podnetworkchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.builder    Registering a validating webhook    {"GVK": "chaos-mesh.org/v1alpha1, Kind=PodNetworkChaos", "path": "/validate-chaos-mesh-org-v1alpha1-podnetworkchaos"}
2020-11-26T12:13:52.579Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/validate-chaos-mesh-org-v1alpha1-podnetworkchaos"}
2020-11-26T12:13:52.579Z    INFO    setup    Setting up webhook server
2020-11-26T12:13:52.579Z    INFO    inject-webhook    Creating Kubernetes client to talk to the api-server
2020-11-26T12:13:52.580Z    INFO    inject-webhook    Created ConfigMap watcher    {"apiserver": "https://10.96.0.1:443", "template namespaces": "chaos-testing", "template labels": {"app.kubernetes.io/component":"template"}, "config labels": {"app.kubernetes.io/component":"webhook"}}
2020-11-26T12:13:52.580Z    INFO    controller-runtime.webhook    registering webhook    {"path": "/inject-v1-pod"}
2020-11-26T12:13:52.580Z    INFO    setup    Starting manager
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "stresschaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "stresschaos"}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "networkchaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "podchaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "iochaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "timechaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "podchaos"}
2020-11-26T12:13:52.581Z    INFO    setup    Launching watcher for ConfigMaps
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "networkchaos"}
2020-11-26T12:13:52.582Z    INFO    inject-webhook    Watching for ConfigMaps for changes    {"template namespace": "chaos-testing", "labels": {"app.kubernetes.io/component":"webhook"}}
2020-11-26T12:13:52.582Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "iochaos"}
2020-11-26T12:13:52.581Z    INFO    controller-runtime.controller    Starting EventSource    {"controller": "kernelchaos", "source": "kind source: /, Kind="}
2020-11-26T12:13:52.582Z    INFO    controller-runtime.manager    starting metrics server    {"path": "/metrics"}
2020-11-26T12:13:52.582Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "kernelchaos"}
2020-11-26T12:13:52.582Z    INFO    controller-runtime.webhook.webhooks    starting webhook server
2020-11-26T12:13:52.583Z    INFO    controller-runtime.certwatcher    Updated current TLS certificate
2020-11-26T12:13:52.583Z    INFO    controller-runtime.webhook    serving webhook server    {"host": "", "port": 9443}
2020-11-26T12:13:52.584Z    INFO    controller-runtime.certwatcher    Starting certificate watcher
2020-11-26T12:13:52.682Z    INFO    controller-runtime.controller    Starting workers    {"controller": "iochaos", "worker count": 1}
2020-11-26T12:13:52.682Z    INFO    controller-runtime.controller    Starting workers    {"controller": "podchaos", "worker count": 1}
2020-11-26T12:13:52.682Z    INFO    controller-runtime.controller    Starting workers    {"controller": "networkchaos", "worker count": 1}
2020-11-26T12:13:52.683Z    INFO    controller-runtime.controller    Starting workers    {"controller": "kernelchaos", "worker count": 1}
2020-11-26T12:13:52.683Z    INFO    controller-runtime.controller    Starting workers    {"controller": "stresschaos", "worker count": 1}
2020-11-26T12:13:52.683Z    INFO    controller-runtime.controller    Starting Controller    {"controller": "timechaos"}
2020-11-26T12:13:52.684Z    INFO    controller-runtime.controller    Starting workers    {"controller": "timechaos", "worker count": 1}

...

2020-11-26T12:14:17.733Z    ERROR    controllers.NetworkChaos    error while creating podnetworkchaos    {"reconciler": "networkchaos", "action": "partition", "error": "admission webhook \"vpodnetworkchaos.kb.io\" denied the request: rpc error: code = Unknown desc = error code: exit status 2, msg: Error: Specified qdisc not found.\n"}
github.com/go-logr/zapr.(*zapLogger).Error
    /root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/controllers/networkchaos/podnetworkmanager.(*PodNetworkManager).Commit.func1.1
    /src/controllers/networkchaos/podnetworkmanager/podnetworkmanager.go:125
k8s.io/client-go/util/retry.OnError.func1
    /root/go/pkg/mod/k8s.io/client-go@v0.17.0/util/retry/util.go:51
k8s.io/apimachinery/pkg/util/wait.ExponentialBackoff
    /root/go/pkg/mod/k8s.io/apimachinery@v0.17.1-beta.0/pkg/util/wait/wait.go:292
k8s.io/client-go/util/retry.OnError
    /root/go/pkg/mod/k8s.io/client-go@v0.17.0/util/retry/util.go:50
k8s.io/client-go/util/retry.RetryOnConflict
    /root/go/pkg/mod/k8s.io/client-go@v0.17.0/util/retry/util.go:104
github.com/chaos-mesh/chaos-mesh/controllers/networkchaos/podnetworkmanager.(*PodNetworkManager).Commit.func1
    /src/controllers/networkchaos/podnetworkmanager/podnetworkmanager.go:83
golang.org/x/sync/errgroup.(*Group).Go.func1
    /root/go/pkg/mod/golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:57
2020-11-26T12:14:17.733Z    ERROR    controllers.NetworkChaos    error while updating    {"reconciler": "networkchaos", "action": "partition", "error": "admission webhook \"vpodnetworkchaos.kb.io\" denied the request: rpc error: code = Unknown desc = error code: exit status 2, msg: Error: Specified qdisc not found.\n"}
github.com/go-logr/zapr.(*zapLogger).Error
    /root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/controllers/networkchaos/podnetworkmanager.(*PodNetworkManager).Commit.func1
    /src/controllers/networkchaos/podnetworkmanager/podnetworkmanager.go:140
golang.org/x/sync/errgroup.(*Group).Go.func1
    /root/go/pkg/mod/golang.org/x/sync@v0.0.0-20190911185100-cd5d95a43a6e/errgroup/errgroup.go:57
```

_**ChaosDaemon Logs**_
```
2020-11-26T12:13:51.835Z    INFO    chaos-daemon    grant access to /dev/fuse
2020-11-26T12:13:51.837Z    INFO    chaos-daemon-server    Starting grpc endpoint    {"address": "0.0.0.0:31767", "runtime": "docker"}
2020-11-26T12:13:51.837Z    INFO    chaos-daemon-server    Starting http endpoint    {"address": "0.0.0.0:31766"}
Chaos-daemon Version: version.Info{GitVersion:"v1.0.2", GitCommit:"1eede0221a0c3ac2add73dec710fecacaad0958c", BuildDate:"2020-11-09T05:56:41Z", GoVersion:"go1.14.6", Compiler:"gc", Platform:"linux/amd64"}
2020-11-26T12:14:17.753Z    INFO    chaos-daemon-server    flush ipset    {"request": "container_id:\"docker://ce819fd5232c8ed5a4ea0341547e61b8694451849e4abf8d9bcbff73df28dcdc\" "}
2020-11-26T12:14:17.753Z    INFO    chaos-daemon-server    flush ipset    {"request": "container_id:\"docker://d85589e6871cbc81aa2c7bbed86816fe468576f37a5dd7e28e503d7997780b0b\" "}
2020-11-26T12:14:17.759Z    INFO    chaos-daemon-server    Set iptables chains    {"request": "container_id:\"docker://d85589e6871cbc81aa2c7bbed86816fe468576f37a5dd7e28e503d7997780b0b\" "}
2020-11-26T12:14:17.759Z    INFO    chaos-daemon-server    Set iptables chains    {"request": "container_id:\"docker://ce819fd5232c8ed5a4ea0341547e61b8694451849e4abf8d9bcbff73df28dcdc\" "}
2020-11-26T12:14:17.760Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -N CHAOS-INPUT"}
2020-11-26T12:14:17.760Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -N CHAOS-INPUT"}
2020-11-26T12:14:17.768Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -F CHAOS-INPUT"}
2020-11-26T12:14:17.768Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -F CHAOS-INPUT"}
2020-11-26T12:14:17.770Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -S INPUT"}
2020-11-26T12:14:17.770Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -S INPUT"}
2020-11-26T12:14:17.773Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -N CHAOS-OUTPUT"}
2020-11-26T12:14:17.773Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -N CHAOS-OUTPUT"}
2020-11-26T12:14:17.775Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -F CHAOS-OUTPUT"}
2020-11-26T12:14:17.775Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -F CHAOS-OUTPUT"}
2020-11-26T12:14:17.777Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- iptables -w -S OUTPUT"}
2020-11-26T12:14:17.778Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- iptables -w -S OUTPUT"}
2020-11-26T12:14:17.782Z    INFO    chaos-daemon-server    handling tc request    {"tcs": "container_id:\"docker://ce819fd5232c8ed5a4ea0341547e61b8694451849e4abf8d9bcbff73df28dcdc\" "}
2020-11-26T12:14:17.783Z    INFO    chaos-daemon-server    handling tc request    {"tcs": "container_id:\"docker://d85589e6871cbc81aa2c7bbed86816fe468576f37a5dd7e28e503d7997780b0b\" "}
2020-11-26T12:14:17.784Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- tc qdisc del dev eth0 root"}
2020-11-26T12:14:17.785Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- tc qdisc del dev eth0 root"}
2020-11-26T12:14:17.790Z    INFO    chaos-daemon-server    adding prio    {"parent": 0}
2020-11-26T12:14:17.791Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/18760/ns/net -- tc qdisc add dev eth0 root handle 1: prio bands 3 priomap 1 2 2 2 1 2 0 0 1 1 1 1 1 1 1 1"}
2020-11-26T12:14:17.791Z    INFO    chaos-daemon-server    adding prio    {"parent": 0}
2020-11-26T12:14:17.791Z    INFO    background-process-manager    build command    {"command": "nsenter -n/proc/21341/ns/net -- tc qdisc add dev eth0 root handle 1: prio bands 3 priomap 1 2 2 2 1 2 0 0 1 1 1 1 1 1 1 1"}
2020-11-26T12:14:17.794Z    ERROR    chaos-daemon-server    error while adding prio    {"error": "error code: exit status 2, msg: Error: Specified qdisc not found.\n"}
github.com/go-logr/zapr.(*zapLogger).Error
    /root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon.(*daemonServer).SetTcs
    /src/pkg/chaosdaemon/tc_server.go:140
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler.func1
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1729
github.com/grpc-ecosystem/go-grpc-prometheus.(*ServerMetrics).UnaryServerInterceptor.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-prometheus@v1.2.0/server_metrics.go:107
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/chaos-mesh/chaos-mesh/pkg/utils.TimeoutServerInterceptor
    /src/pkg/utils/grpc.go:71
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:34
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1731
google.golang.org/grpc.(*Server).processUnaryRPC
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1024
google.golang.org/grpc.(*Server).handleStream
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1313
google.golang.org/grpc.(*Server).serveStreams.func1.1
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:722
2020-11-26T12:14:17.796Z    ERROR    chaos-daemon-server    error while adding prio    {"error": "error code: exit status 2, msg: Error: Specified qdisc not found.\n"}
github.com/go-logr/zapr.(*zapLogger).Error
    /root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon.(*daemonServer).SetTcs
    /src/pkg/chaosdaemon/tc_server.go:140
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler.func1
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1729
github.com/grpc-ecosystem/go-grpc-prometheus.(*ServerMetrics).UnaryServerInterceptor.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-prometheus@v1.2.0/server_metrics.go:107
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/chaos-mesh/chaos-mesh/pkg/utils.TimeoutServerInterceptor
    /src/pkg/utils/grpc.go:71
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:34
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1731
google.golang.org/grpc.(*Server).processUnaryRPC
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1024
google.golang.org/grpc.(*Server).handleStream
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1313
google.golang.org/grpc.(*Server).serveStreams.func1.1
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:722
```

**Comment by jujoramos** [comment]
Even though the error is thrown when deploying the `network-split` scenario to an internal `kubernetes` cluster managed by the company I work for, I've just been to reproduce a really similar issue (the error is exactly the same) using `minikube` and a simple `StatefulSet`.
I'm adding the details here, hopefully it will make the troubleshooting easier :-).

**Launch Minikube**
```
minikube start --cpus 4 --memory "8192mb"
😄  minikube v1.12.1 on Darwin 10.15.7
✨  Using the virtualbox driver based on user configuration
👍  Starting control plane node minikube in cluster minikube
🔥  Creating virtualbox VM (CPUs=4, Memory=8192MB, Disk=20000MB) ...
🐳  Preparing Kubernetes v1.18.3 on Docker 19.03.12 ...
🔎  Verifying Kubernetes components...
🌟  Enabled addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube"
```

**Install Chaos Mesh**
```
curl -sSL https://mirrors.chaos-mesh.org/v1.0.2/install.sh | bash
Install Chaos Mesh chaos-mesh
customresourcedefinition.apiextensions.k8s.io/iochaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/kernelchaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/networkchaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/podchaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/podiochaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/podnetworkchaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/stresschaos.chaos-mesh.org created
customresourcedefinition.apiextensions.k8s.io/timechaos.chaos-mesh.org created
namespace/chaos-testing created
serviceaccount/chaos-controller-manager created
secret/chaos-mesh-webhook-certs created
clusterrole.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-target-namespace created
clusterrole.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-cluster-level created
clusterrolebinding.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-cluster-level created
clusterrolebinding.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-target-namespace created
role.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-control-plane created
rolebinding.rbac.authorization.k8s.io/chaos-mesh:chaos-controller-manager-control-plane created
service/chaos-dashboard created
service/chaos-mesh-controller-manager created
daemonset.apps/chaos-daemon created
deployment.apps/chaos-dashboard created
deployment.apps/chaos-controller-manager created
mutatingwebhookconfiguration.admissionregistration.k8s.io/chaos-mesh-mutation created
validatingwebhookconfiguration.admissionregistration.k8s.io/chaos-mesh-validation created
Waiting for pod running
chaos-controller-manager-59886dfb4d-54wps   0/1   ContainerCreating   0     11s
Waiting for pod running
chaos-controller-manager-59886dfb4d-54wps   0/1   ContainerCreating   0     21s
Waiting for pod running
Chaos Mesh chaos-mesh is installed successfully
```

**Deploy a simple StatefulSet**
```
cat << EOF | kubectl apply -f -
apiVersion: apps/v1
kind: StatefulSet
metadata:
  name: web
spec:
  selector:
    matchLabels:
      app: nginx
  serviceName: "nginx"
  replicas: 3
  template:
    metadata:
      labels:
        app: nginx
    spec:
      terminationGracePeriodSeconds: 10
      containers:
      - name: nginx
        image: k8s.gcr.io/nginx-slim:0.8
        ports:
        - containerPort: 80
          name: web
EOF

statefulset.apps/web created
```

**Deploy Network-Split Scenario**
```
cat << EOF | kubectl apply -f -
apiVersion: chaos-mesh.org/v1alpha1
kind: NetworkChaos
metadata:
  name: network-partition-example
  namespace: default
spec:
  action: partition
  mode: all
  selector:
    pods:
      default:
        - web-0
  direction: both
  target:
    mode: all
    selector:
      pods:
        default:
          - web-1
          - web-2
  duration: '120s'
  scheduler:
    cron: '@every 240s'
EOF

networkchaos.chaos-mesh.org/network-partition-example created
```

**Error logs shown on chaos-daemon**
```
2020-11-26T15:35:41.443Z    ERROR    chaos-daemon-server    error while adding prio    {"error": "error code: exit status 2, msg: Error: Specified qdisc not found.\n"}
github.com/go-logr/zapr.(*zapLogger).Error
    /root/go/pkg/mod/github.com/go-logr/zapr@v0.1.0/zapr.go:128
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon.(*daemonServer).SetTcs
    /src/pkg/chaosdaemon/tc_server.go:140
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler.func1
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1729
github.com/grpc-ecosystem/go-grpc-prometheus.(*ServerMetrics).UnaryServerInterceptor.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-prometheus@v1.2.0/server_metrics.go:107
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/chaos-mesh/chaos-mesh/pkg/utils.TimeoutServerInterceptor
    /src/pkg/utils/grpc.go:71
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1.1.1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:25
github.com/grpc-ecosystem/go-grpc-middleware.ChainUnaryServer.func1
    /root/go/pkg/mod/github.com/grpc-ecosystem/go-grpc-middleware@v1.2.0/chain.go:34
github.com/chaos-mesh/chaos-mesh/pkg/chaosdaemon/pb._ChaosDaemon_SetTcs_Handler
    /src/pkg/chaosdaemon/pb/chaosdaemon.pb.go:1731
google.golang.org/grpc.(*Server).processUnaryRPC
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1024
google.golang.org/grpc.(*Server).handleStream
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:1313
google.golang.org/grpc.(*Server).serveStreams.func1.1
    /root/go/pkg/mod/google.golang.org/grpc@v1.26.0/server.go:722
```

**Comment by YangKeao** [comment]
Thanks for your report. I will try to reproduce and fix this bug right now.

**Comment by YangKeao** [comment]
Please upgrade to minikube >= 1.12.2. I have added some kernel configuration for their image in this [PR](https://github.com/kubernetes/minikube/pull/8670) and it's merged into minikube after 1.12.2

**Comment by jujoramos** [comment]
Thanks for the update @YangKeao !!. I've tried with the latest `minikube` version (`v1.15.1`) and it works just fine.
The problem remains on the `VMware Tanzu Kubernetes Grid` we use for testing purposes, however... I guess I'll need to check whether I can change the kernel configuration or something like you did for the `minikube`image.
